src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -232,8 +232,10 @@ public boolean clickShowNewInstructorFormButton() {
         return isFormShownCorrectly;
     }
     
-    public boolean clickOnAccessLevelViewDetails(String role) {
-        WebElement viewDetailsLink = browser.driver.findElement(By.cssSelector(
+    public boolean clickOnAccessLevelViewDetails(String formId, String role) {
```

Comment:
I'm not sure if passing in a CSS id is the usual way of doing things. The more common form is passing in an integer index (which will require refactoring to a sentinel value for new instructor), or if the element is unique enough, there will be a `@FindBy` WebElement.